.env.example

```
@@ -1 +1 @@
-REACT_APP_RAPID_API_KEY = <KEY>
+REACT_APP_RAPID_API_KEY = AIzaSyD4WKhhYFE1YnlNCfPvSrmeNRdHaMnD8wI
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)